packages/workbox-background-sync/demo/sw.js

```diff
@@ -1,8 +1,8 @@
 /* global goog*/
 importScripts(
 	'../build/background-sync-queue.js',
-	'../../sw-routing/build/sw-routing.js',
-	'../../sw-runtime-caching/build/sw-runtime-caching.js'
+	'../../workbox-routing/build/workbox-routing.js',
+	'../../workbox-runtime-caching/build/workbox-runtime-caching.js'
```

Comment:
shouldn't Line 17: `new goog.backgroundSync.QueuePlugin` should also be changed?